src/utils/contents.js

```diff
@@ -50,6 +54,8 @@ export function encodeContenthash(text) {
         encoded = '0x' + contentHash.fromIpfs(content)
       } else if (contentType === 'bzz') {
         encoded = '0x' + contentHash.fromSwarm(content)
+      } else if (contentType === 'onion') {//In the future insert here general encoding given by contentHash library
+        encoded = '0x' + contentHash.fromOnion(content)
```

Comment:
this should be `contentHash.encode('onion', content);`

Reply:
done